.travis.yml

```diff
@@ -34,8 +34,8 @@ before_install:
     - echo -e "machine github.com\n  login $GH_ACCESS_TOKEN" >> ~/.netrc # Login to GitHub
     - echo -e "machine github.ibm.com\n  login $GHE_ACCESS_TOKEN" >> ~/.netrc # Login to GitHub Enterprise
 install:
-    - pip install tox
-script: make test
+    - pip install tox pipenv
+script: make setup-trivy && make trivy-scan-python-vulnerabilities && make test
 cache:
     directories:
```

Comment:
For every branch build, set up Trivy and run a Trivy vulnerability scan before  running tests